templates/components/carousel.html

```diff
@@ -77,5 +77,11 @@
             }'>
             {{lang 'carousel.playPauseButtonPause'}}
         </button>
+    {{/and}}
+    {{#if arrows}}
+    <span 
+        class="carousel-content-change-message aria-description--hidden" role="status">
+        {{lang 'carousel.contentAnnounceMessage'}} {{carousel.slides.length}}
+    </span>
```

Comment:
also - can we check slides quantity on the same level with arrows existence? because if we have just one slide we do not need this html

Reply:
good point, I've missed it by checking only on arrows presentation in theme settings.